packages/plugin-lighthouse/src/lib/lighthouse-plugin.integration.test.ts

```diff
@@ -10,3 +10,15 @@ describe('lighthousePlugin', () => {
     expect(pluginConfig.groups).toHaveLength(5);
   });
 });
+
+describe('getRunner', () => {
+  it('should create and execute runner correctly', async () => {
+    // onlyAudits is used to reduce test time
+    const runner = getRunner('https://example.com', {
+      onlyAudits: ['is-on-https'],
+    });
+    await expect(runner(() => void 0)).resolves.toEqual([
+      expect.objectContaining({ slug: 'is-on-https' }),
+    ]);
+  });
+}, 70_000);
```

Comment:
Do we really need such a long timeout? :open_mouth:

Reply:
Only on windows, but I'm researching tricks to spin up chromium with a cached website and taking away internet delay or flakiness. 🕵️

My plan is to ship a solution with the e2e test PR that is missing.

Reply:
I will close this here and open a issue on the current research and document it. 
I tried to get it into this PR but it seemed not possible to deliver a good solution in this PR.

Feel free to reopen it.

Reply:
If it's only needed for Windows, then setting the timeout based on OS could be a practical workaround - `platform() === 'win32' ? 70_000 : <something smaller>` :thinking: